src/Traits/Authenticatable.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace TCG\Voyager\Traits;
+
+use Illuminate\Foundation\Auth\User;
+
+class Authenticatable extends User
```

Comment:
This is a class, not a trait, and shouldn't be a trait.  Please change the namespace.

Reply:
Done